Source/WebKit/NetworkProcess/NetworkSession.cpp

```diff
@@ -680,6 +680,11 @@ SWServer& NetworkSession::ensureSWServer()
     return *m_swServer;
 }
 
+Ref<WebCore::SWServer> NetworkSession::protectedSWServer()
```

Comment:
Seems like this should be called `ensureProtectedSWServer()`. Otherwise, we have no idea that this constructs the server if needed.